src/Microsoft.ServiceFabric.Actors/Runtime/ActorTypeInformation.cs

```diff
@@ -125,10 +125,10 @@ public static ActorTypeInformation Get(Type actorType)
                 actorServiceName = actorServiceAttr.Name;
             }
 
-            // get all actor interfaces
+            // Gets all actor interfaces.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.